src/routes/exportRoutes.js

```
@@ -0,0 +1,172 @@
+      details: { type, timestamp: new Date().toISOString() }
+    });
+
+    let data = [];
+    let filename = '';
+
+    switch (type) {
+      case 'bookings':
+        const { data: bookings, error: bookingError } = await supabaseAdmin
+          .from('bookings')
+          .select('*')
+          .order('created_at', { ascending: false });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)